ruby/read_file.rb

```diff
@@ -0,0 +1,23 @@
+"""Read a file from file path (given as a command line arg),
+print line by line with line numbers
+"""
+if ARGV.length != 1
+    puts "Make sure to enter the filepath!"
+    exit(1)
+  end
+
+  file_path = ARGV[0]
+
+  begin
+    i = 1
+    File.open(file_path, 'r') do |f|
+      f.each_line do |line|
```

Comment:
I'd probably do 
```ruby 
f.each_line.with_index do |line, i|
  puts "#{i+1} #{line.upcase}"
end
```
The `i+1` is b/c `with_index` will start `i` at 0

Reply:
Nice!